vllm/v1/engine/core_client.py

```diff
@@ -385,7 +385,7 @@ def close_sockets_and_tasks():
                         with contextlib.suppress(Exception):
                             task.cancel()
 
-            if in_loop(loop):
+            if loop is not None and in_loop(loop):
                 close_sockets_and_tasks()
             elif loop and not loop.is_closed():
                 loop.call_soon_threadsafe(close_sockets_and_tasks)
```

Comment:
Maybe this instead of checking if the `loop` is not None twice?

```suggestion
            if loop is not None:
                if in_loop(loop):
                    close_sockets_and_tasks()
                elif not loop.is_closed():
                    loop.call_soon_threadsafe(close_sockets_and_tasks)
```

Reply:
Fixed, thanks!